docs/build-docs.sh

```diff
@@ -15,6 +15,6 @@ sphinx-build -W -b html -D html_title="$EXTENSION_NAME" . build || exit 1
 sphinx-build -W -v -b coverage . build/coverage || exit 1
 
 # Remove copied files. This ensures reusability.
-rm -r _static 
+rm -r _static
```

Comment:
This file will be deleted eventually as all of the build process will be handled by the justfile. Similarly for `install.sh`